exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -55,6 +55,15 @@ export const REPORT_SUMMARY_OPTION_HELP = {
   name: '--report-summary',
 }
 
+export const COLLAPSE_OUTPUT_OPTION = {
+  'collapse-output': Boolean,
+}
+
+export const COLLAPSE_OUTPUT_OPTION_HELP = {
+  description: 'Collapse the output of --parallel by disabling the --stream option, which is enabled by default when you run --parallel.',
+  name: '--collapse-output',
+}
+
```

Comment:
should there be a new option? Wouldn't `--no-stream` work too?

Reply:
--no-stream works great. I didn't know that you can use --no- prefix in order to disable boolean options in pnpm. Very cool!
However, when running in a single project, the output is still not collapsed:

![image](https://github.com/pnpm/pnpm/assets/79661007/a7c37768-f1e2-41cc-b482-e31a30a926de)

The --collapse-output option disables the --stream option (like --no-stream), but also sets the "stdio" variable to "pipe" in this file: exec/plugin-commands-script-runners/src/runRecursive.ts. As a result, an output is collapsed:

![image](https://github.com/pnpm/pnpm/assets/79661007/fad1af18-6443-451b-b029-863ddc48ce9b)